frontend/src/components/buttons/success button/SuccessButton.js

```diff
@@ -0,0 +1,17 @@
+import BaseButton from "../BaseButton.js";
+
+export default class SuccessButton extends BaseButton {
+  constructor(args) {
+    super(args);
+
+    this._setStyles();
+  }
+
+  _setStyles() {
+    this._button.classList.add('btn', 'btn-success', 'btn-lg', 'start');
```

Comment:
It can be store in const in base class. Why do we need all these subclasses just for different themes?